sdk/eventhub/event-hubs/test/receiver.spec.ts

```diff
@@ -61,10 +76,25 @@ describe("EventHub Receiver", function(): void {
 
   describe("with partitionId 0 as number", function(): void {
     it("should not throw an error", async function(): Promise<void> {
-      receiver = client.createConsumer(EventHubConsumerClient.defaultConsumerGroupName, 0 as any, {
-        sequenceNumber: 0
+      let subscription: Subscription | undefined;
+      await new Promise((resolve, reject) => {
+        subscription = consumerClient.subscribe(
+          0 as any,
```

Comment:
Rather than casting to any you can use this construct now:

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#-ts-expect-error-comments

(I did this _somewhere_ in the code base and it's a nice way of documenting it).

Reply:
Cool, done with https://github.com/Azure/azure-sdk-for-js/pull/9521/commits/9692f581f91242337020d9fdafb980b0968412f5